Content.Server/Chat/ChatManager.cs

```diff
@@ -115,15 +120,31 @@ public void EntitySay(IEntity source, string message)
             var pos = source.Transform.Coordinates;
             var clients = _playerManager.GetPlayersInRange(pos, VoiceRange).Select(p => p.ConnectedClient);
 
+            if (message.StartsWith(';'))
+            {
+                message = message.Substring(1);
```

Comment:
Maybe trim the start of the string? Many people will write "; hello" or ";hello" and we probably don't want the space there.